packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -1,440 +1,21 @@
 @import 'tailwindcss/base';
```

Comment:
This file is now very small, I recommend [looking at it directly](https://github.com/coveo/ui-kit/blob/KIT-873/packages/atomic/src/components/atomic-result/atomic-result.pcss).